plat/nxp/common/bl2_el3_setup.c

```diff
@@ -0,0 +1,328 @@
+/*
+ * Copyright 2018-2019 NXP
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ *
+ */
+#include <assert.h>
+
+#include <common/bl_common.h>
+#include <common/desc_image_load.h>
+#include <common/tbbr/tbbr_img_def.h>
+#include <drivers/console.h>
+
+#include <mmu_def.h>
+#include <sfp.h>
+#include <plat_common.h>
+
+
+#define BL2_END                    (unsigned long)(&__BL2_END__)
```

Comment:
Use the definitions in ``bl_common.h`` rather than creating definitions here. If needed, add it to ``bl_common.h``.